### ISSUE TITLE
WHOIS v1.20 Issue

### ISSUE THREAD

**Issue by alanchau** [MicrosoftDocs/sysinternals#51]
```
C:\Windows\System32>whois godaddy.com

Whois v1.20 - Domain information lookup
Copyright (C) 2005-2017 Mark Russinovich
Sysinternals - www.sysinternals.com

Connecting to COM.whois-servers.net...

WHOIS Server: whois.godaddy.com
   Registrar URL: http://www.godaddy.com
   Updated Date: 2014-04-09T04:15:36Z
   Creation Date: 1999-03-02T05:00:00Z
   Registry Expiry Date: 2021-11-01T11:59:59Z
   Registrar: GoDaddy.com, LLC
   Registrar IANA ID: 146
   Registrar Abuse Contact Email: abuse@godaddy.com
   Registrar Abuse Contact Phone: 480-624-2505
   Domain Status: clientDeleteProhibited https://icann.org/epp#clientDeleteProhibited
   Domain Status: clientRenewProhibited https://icann.org/epp#clientRenewProhibited
   Domain Status: clientTransferProhibited https://icann.org/epp#clientTransferProhibited
   Domain Status: clientUpdateProhibited https://icann.org/epp#clientUpdateProhibited
   Domain Status: serverDeleteProhibited https://icann.org/epp#serverDeleteProhibited
   Domain Status: serverTransferProhibited https://icann.org/epp#serverTransferProhibited
   Domain Status: serverUpdateProhibited https://icann.org/epp#serverUpdateProhibited
   Name Server: A1-245.AKAM.NET
   Name Server: A11-64.AKAM.NET
   Name Server: A20-65.AKAM.NET
   Name Server: A6-66.AKAM.NET
   Name Server: A8-67.AKAM.NET
   Name Server: A9-67.AKAM.NET
   Name Server: CNS1.GODADDY.COM
   Name Server: CNS2.GODADDY.COM
   Name Server: CNS3.GODADDY.COM
   DNSSEC: unsigned
   URL of the ICANN Whois Inaccuracy Complaint Form: https://www.icann.org/wicf/
>>> Last update of whois database: 2018-01-09T06:23:04Z <<<

For more information on Whois status codes, please visit https://icann.org/epp

NOTICE: The expiration date displayed in this record is the date the
registrar's sponsorship of the domain name registration in the registry is
currently set to expire. This date does not necessarily reflect the expiration
date of the domain name registrant's agreement with the sponsoring
registrar.  Users may consult the sponsoring registrar's Whois database to
view the registrar's reported date of expiration for this registration.

TERMS OF USE: You are not authorized to access or query our Whois
database through the use of electronic processes that are high-volume and
automated except as reasonably necessary to register domain names or
modify existing registrations; the Data in VeriSign Global Registry
Services' ("VeriSign") Whois database is provided by VeriSign for
information purposes only, and to assist persons in obtaining information
about or related to a domain name registration record. VeriSign does not
guarantee its accuracy. By submitting a Whois query, you agree to abide
by the following terms of use: You agree that you may use this Data only
for lawful purposes and that under no circumstances will you use this Data
to: (1) allow, enable, or otherwise support the transmission of mass
unsolicited, commercial advertising or solicitations via e-mail, telephone,
or facsimile; or (2) enable high volume, automated, electronic processes
that apply to VeriSign (or its computer systems). The compilation,
repackaging, dissemination or other use of this Data is expressly
prohibited without the prior written consent of VeriSign. You agree not to
use electronic processes that are automated and high-volume to access or
query the Whois database except as reasonably necessary to register
domain names or modify existing registrations. VeriSign reserves the right
to restrict your access to the Whois database in its sole discretion to ensure
operational stability.  VeriSign may restrict or terminate your access to the
Whois database for failure to abide by these terms of use. VeriSign
reserves the right to modify these terms at any time.

The Registry database contains ONLY .COM, .NET, .EDU domains and
Registrars.

Connecting to whois.godaddy.com...

Domain Name: GODADDY.COM
Registrar URL: http://www.godaddy.com
Registrant Name: Domain Administrator
Registrant Organization: Go Daddy Operating Company, LLC
Name Server: CNS1.GODADDY.COM
Name Server: CNS2.GODADDY.COM
Name Server: CNS3.GODADDY.COM
Name Server: A11-64.AKAM.NET
Name Server: A1-245.AKAM.NET
Name Server: A20-65.AKAM.NET
Name Server: A6-66.AKAM.NET
Name Server: A8-67.AKAM.NET
Name Server: A9-67.AKAM.NET
DNSSEC: unsigned

For complete domain details go to:
http://who.godaddy.com/whoischeck.aspx?domain=GODADDY.COM

The data contained in GoDaddy.com, LLC's WhoIs database,
while believed by the company to be reliable, is provided "as is"
with no guarantee or warranties regarding its accuracy.  This
information is provided for the sole purpose of assisting you
in obtaining information about domain name registration records.
Any use of this data for any other purpose is expressly forbidden without the prior written
permission of GoDaddy.com, LLC.  By submitting an inquiry,
you agree to these terms of usage and limitations of warranty.  In particular,
you agree not to use this data to allow, enable, or otherwise make possible,
dissemination or collection of this data, in part or in its entirety, for any
purpose, such as the transmission of unsolicited advertising and
and solicitations of any kind, including spam.  You further agree
not to use this data to enable high volume, automated or robotic electronic
processes designed to collect or compile this data for any purpose,
including mining this data for your own personal or commercial purposes.

Please note: the registrant of the domain name is specified
in the "registrant" section.  In most cases, GoDaddy.com, LLC
is not the registrant of domain names listed in this database.

Connecting to ODADDY.COM...
```

When doing a WHOIS lookup for godaddy.com for example, it properly redirects to whois.godaddy.com but then for some reason does a redirect to "odaddy.com".

**Comment by vfrazao-ns1** [comment]
I actually experience different (also bad) behaviour when using the whois utility to find `godaddy.com` - the program seems to go into an infinite loop and will only stop if you press ctrl+c. It seems to perpetually try to connect to the same whois server:
```shell
PS C:\Users\luiz_> whois godaddy.com | Select-String "Connecting to"

Connecting to COM.whois-servers.net...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
Connecting to whois.godaddy.com...
```

**Comment by lukekim** [comment]
Thanks for reporting this, it should be fixed in v1.21.
